pylint/checkers/format.py

```diff
@@ -499,20 +499,15 @@ def process_tokens(self, tokens):
 
         line_num -= 1  # to be ok with "wc -l"
         if line_num > self.linter.namespace.max_module_lines:
-            # Get the line where the too-many-lines (or its message id)
-            # was disabled or default to 1.
-            message_definition = self.linter.msgs_store.get_message_definitions(
-                "too-many-lines"
-            )[0]
-            names = (message_definition.msgid, "too-many-lines")
-            line = next(
-                filter(None, (self.linter._pragma_lineno.get(name) for name in names)),
-                1,
-            )
+            # Get the line where the too-many-lines (or its message id) was disabled
+            lineno = self.linter._pragma_lineno.get("C0302")
```

Comment:
I think using `get_message_definitions` here makes this more robust for any future changes to message id (going from 4 to 5 digits, old names, split-up checkers and new id's).

Reply:
It would also be vastly less performant: there's 4/5 calls to the message store to get the value, in process_token for each pylint run, for each user to get a value we already have that we'll maybe change once or twice in the future. (Also it did not handle old names).

Reply:
It's cached with a `maxsize=None` so performance should be okay after a first call right?

Reply:
Right, let me do some benchmark / handling of old_names then.